src/FixedDataTable.js

```diff
@@ -577,9 +577,9 @@ class FixedDataTable extends React.Component {
       this._didScrollStart();
     }
 
-    // Cancel any pending debounced scroll handling and handle immediately.
-    this._didScrollStop.reset();
-    this._didScrollStopSync();
+    if (!this.props.scrollJumped && nextProps.scrollJumped) {
+      this._didScrollJump(nextProps);
```

Comment:
`nextProps` contains the updated props. I'm suprised the `_didScrollStart` call above this doesn't use the updated props, is it not needed?.